.github/ISSUE_TEMPLATE/collection_proposal.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Propose a new collection
+about: Use this ticket when you are submitting new collection for review.
```

Comment:
```suggestion
about: Use this ticket when you are submitting new a collection for the site. If this doesn’t look right, [choose a different type](https://github.com/GoogleChrome/web.dev/issues/new/choose).
```
Should the other ticket read "to propose a new post/codelab" instead of "content" for clarity?

Reply:
I think github automatically inserts that "If this doesn't look right..." text for us. I looked and none of the other templates contain that text.